netket/models/slater.py

```diff
@@ -0,0 +1,112 @@
+# Copyright 2023 The NetKet Authors - All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import flax.linen as nn
+import jax.numpy as jnp
+
+from functools import partial
+
+from netket.experimental.hilbert import SpinOrbitalFermions
+from netket.utils.types import NNInitFunc, DType
+from netket.nn.masked_linear import default_kernel_init
+from netket import jax as nkjax
+
+
+def _log_det(A):
```

Comment:
I'd move this to nkjax.

Reply:
Then I'd rename to  `logdet`  for consistency with `logsumexp` and numpy naming that we tend to follow.